src/ui/summary-ui-handler.ts

```diff
@@ -23,6 +22,7 @@ import { Ability } from "../data/ability.js";
 import i18next from "i18next";
 import {modifierSortFunc} from "../modifier/modifier";
 import { PlayerGender } from "#enums/player-gender";
+import { Stat, getStatKey, PERMANENT_STATS } from "#app/enums/stat.js";
```

Comment:
```suggestion
import { Stat, PERMANENT_STATS, getStatKey } from "#app/enums/stat.js";
```
NIT: Label types and standardize order